src/page.rs

```diff
@@ -10,6 +10,9 @@ pub struct PageSpec {
     pub redirect: Option<String>,
     #[serde(skip)]
     pub rendered: String,
+    // add struct OGStruct here
+    // it would have fields title, image, url, type
+    //
```

Comment:
This has to be optional struct. Also for future, such approach PR could have included the commented out actual line instead of instruction.

Reply:
We also discussed a trait `ToSEO` with `toSEOwith(&self) -> Option<SEO>` default implementation (that returns None) for types implementing `realm::Page`.

Reply:
Also do a survey of other tags, eg there are twitter specific headers.

Reply:
Maybe we can rename this whole thing from being specific to SEO, to make it HTML Meta specific so we can overwrite favicon, css etc also.